docs/user/dmap_old.md

```diff
@@ -0,0 +1,182 @@
+# Reading and Writing DMap format files
+Please see API documentation on [`DmapRead`](code/DmapRead.md) and [`DmapWrite`](code/DmapWrite.md) for information on class methods and attributes. 
+
+Data Map (DMap) is a binary self-describing format that was developed by Rob Barnes. 
+This format is currently the primary format used by SuperDARN. 
+For more information on DMap please see [RST Documentation](https://radar-software-toolkit-rst.readthedocs.io/en/latest/). 
+
+pyDARN provides two classes to read (*DmapRead*) and write (DmapWrite) DMap formated files. 
+
+!!! Note
+    After pyDARN release 1.0, these modules will be moved to it's own python library. 
+
+## DmapRead 
+
+*DmapRead* reads DMap format files and data streams. 
+Here are some examples on how to use the class:
+
+### Read from a file
+
+```python
+import pydarn
+
+dmap_file = "20180220.C0.rkn.fitacf"
+dmap_read = pydarn.DmapRead(fitacf_file)
+dmap_data = dmap_read.read_records()
+```
+
+Accessing data from pyDARN Dmap structure:
+```python
+# loop over every record in the dmap structure
+for dmap_rec in dmap_data:
+  # check if the beam number is 7  
+  if dmap_rec['bmnum'].value == 7:
+    # print the channel number for that record 
+    # corresponding to beam number 7 
+    print(dmap_rec['channel'].value)
+```
+
+To convert to python dictionary:
+```python
+dmap_dict = pydarn.dmap2dict(dmap_data)
```

Comment:
This raises an attribute error in `pydarn\utils\conversions.py` on line 125 when `.value` is used but the data it is attached to is an integer.